packages/manager/apps/pci-private-registry/src/pages/IAM/IAMModal/IAMModal.tsx

```diff
@@ -181,11 +189,18 @@ export const IAMModal = ({
       <Controller
         name="confirmIAM"
         control={control}
-        render={({ field: { value, onChange }, fieldState: { error } }) => (
+        render={({
+          field: { value, onChange },
+          fieldState: { error, isDirty },
+        }) => (
           <OsdsFormField
             className="mt-6"
             data-testid="confirm-formfield"
-            error={error ? t('common_field:common_field_error_required') : ''}
+            error={
+              error && isDirty
```

Comment:
It seems weird to test if input is dirty, isn't there a mode more suitable to do this ? Something like onBlur ? Or maybe with revalidateMode ?

Reply:
checked together, thanks ;)